exams/pearson/constants.py

```diff
@@ -6,6 +6,23 @@
 
 PEARSON_FILE_TYPE_EAC = 'eac'
 PEARSON_FILE_TYPE_VCDC = 'vcdc'
+PEARSON_FILE_TYPE_EXAM = 'exam'
+PEARSON_FILE_TYPE_CAND = 'cand'
+PEARSON_FILE_TYPE_SURV = 'surv'
+PEARSON_FILE_TYPE_CMNT = 'cmnt'
+PEARSON_FILE_TYPE_SECT = 'sect'
+PEARSON_FILE_TYPE_RESP = 'resp'
+PEARSON_FILE_TYPE_ITEM = 'item'
+
+# these are files we intentionally skip, but treat as if we processed them
+PEARSON_INTENDED_SKIP_FILE_TYPES = (
+    PEARSON_FILE_TYPE_CAND,
+    PEARSON_FILE_TYPE_SURV,
+    PEARSON_FILE_TYPE_CMNT,
+    PEARSON_FILE_TYPE_SECT,
+    PEARSON_FILE_TYPE_ITEM,
+    PEARSON_FILE_TYPE_RESP,
+)
```

Comment:
I'm not familiar with how you'd use a namedtuple in that way, would an `enum` be more appropriate here?